openlogin/tkey-input/index.test.ts

```diff
@@ -0,0 +1,217 @@
+import { expect, Page } from "@playwright/test";
+import { test } from "./index.lib";
+import { useAutoCancel2FASetup, signInWithEmail, findLink } from "../../utils";
+import {
+  useAutoCancelShareTransfer,
+  generateRandomEmail,
+} from "../../utils/index";
+import Mailosaur from "mailosaur";
+import { generate } from "generate-password";
+
+const mailosaur = new Mailosaur(process.env.MAILOSAUR_API_KEY || "");
+
+const testEmail = generateRandomEmail();
+
+const backupEmail = "backup" + generateRandomEmail();
+
+let emailBackupShare = "";
+
+const passwordShare = generate({
+  length: 12,
+  numbers: true,
+  uppercase: true,
+  lowercase: true,
+  strict: true,
+});
+
+let backupPhrase = "";
+
+test.describe.serial("tkey Input test", () => {
+  let page: Page;
+  test.beforeAll(async ({ browser, openloginURL }) => {
+    const context = await browser.newContext();
+    page = await context.newPage();
+    await page.goto(openloginURL);
+    await signInWithEmail(page, testEmail, browser);
+    await mailosaur.messages.deleteAll(process.env.MAILOSAUR_SERVER_ID || ""); // Deleting emails in email server.
```

Comment:
@sidharthWeb3auth Can we do something about `mailosaur.messages.deleteAll` ? because it deletes all emails from entire server. what if other tests are reading from the inbox ?

Reply:
It's doable if we filter emails with a combination of sendTo and subject.
https://mailosaur.com/docs/languages/nodejs/#messagesgetserver-criteria-options

Reply:
Hmm. The other way is to delete the email based on ID. But are we gonna use this mailsour for any other purpose rather than testing it. Also mailsaur supports multiple email servers if in case we need if for any other purpose. wdyt ?

Reply:
@PoojaChopra updated PR